openprescribing/measure_definitions/lyrica.json

```diff
@@ -28,11 +28,13 @@
   "is_cost_based": true,
   "low_is_good": true,
   "numerator_type": "bnf_quantity",
-  "numerator_bnf_codes_query": [
-    "SELECT bnf_code FROM {hscic}.presentation WHERE bnf_code LIKE '0408010AEBB%' AND name LIKE '%_Cap%'--Lyrica capsules"
+  "numerator_bnf_codes_filter": [
+    "0408010AEBB # Lyrica",
+    "name_contains:_Cap "
   ],
   "denominator_type": "bnf_quantity",
-  "denominator_bnf_codes_query": [
-    "SELECT bnf_code FROM {hscic}.presentation WHERE bnf_code LIKE '0408010AE%' AND name LIKE '%_Cap%' --Pregabablin capsules(brands and generic)"
+  "denominator_bnf_codes_filter": [
+    "0408010AE # Pregabablin",
+    "name_contains:_Cap "
```

Comment:
Yeah!  The motivation for doing this *now* is to make is easier to work out which measures are affected by the ODD changes, since in most cases we now don't need to query the prescribing data at all to compute which presentations contribute to a measure.